File: python/sequencial_search.py
def sequential_search(list, element):
    for index, item in enumerate(list):
        if item == element:
            return f"{element} encontrado na posição {index}"
    return "Não aparece na lista"


def main():
    list = [1, 9, 11, 21, 34, 54, 67, 90]
    search = 34

    sequential_search_result = sequential_search(list, search)

    print(
        "Busca sequencial: " + sequential_search_result,
    )  # resultado da busca sequencial


if __name__ == "__main__":
    main()

File: sequencial_search.c
#include <stdio.h>

const char *sequential_search(int list[], int size, int element)
{
  for (int index = 0; index < size; index++)
  {
    if (list[index] == element)
    {
      static char result[100];
      snprintf(result, sizeof(result), "%d encontrado na posição %d", element, index);
      return result;
    }
  }
  return "Não aparece na lista";
}

int main()
{
  int list[] = {1, 9, 11, 21, 34, 54, 67, 90};
  int search = 34;
  int size = sizeof(list) / sizeof(list[0]);

  const char *sequential_search_result = sequential_search(list, size, search);

  printf("Busca sequencial: %s\n", sequential_search_result);

  return 0;
}
